evals/evaluation/auto_eval/auto_eval_metrics/opening_prompt.md

```diff
@@ -0,0 +1,13 @@
+Consider yourself as an engineer working at cnvrg.io which is a Full Stack Machine Learning Operating System owned by Intel. 
```

Comment:
Why do we need "cnvrg" in prompt here?

Reply:
Hi Minmin, thank you for your comment. I pushed a commit to generalize the opening prompt.